services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -410,6 +418,13 @@ public VeniceControllerConfig(VeniceProperties props) {
     this.pushStatusStoreHeartbeatExpirationTimeInSeconds =
         props.getLong(PUSH_STATUS_STORE_HEARTBEAT_EXPIRATION_TIME_IN_SECONDS, TimeUnit.MINUTES.toSeconds(10));
     this.isDaVinciPushStatusStoreEnabled = props.getBoolean(PUSH_STATUS_STORE_ENABLED, false);
+    this.offlinePushMonitorDaVinciPushStatusEnabled =
```

Comment:
Can we consolidate these two configs?

Reply:
It is actually serving different purpose.
For the above one, when it is enabled, the controller will be able to r/w from push status store.
For the below one which is newly added, the collector will consider DVC push status during offline push status check

Reply:
Emm..
When does Controller need to write to push status store?
For incremental push? or to materialize the push status system store at store creation time?

Reply:
There is a call in getOfflinePushStatus to manipulate with inc push status. Although it is not very clear to me what it is doing exactly for now, will need to check. Also there is deleter that deletes push status.

Reply:
Synced offline, decided to leave it for now.